src/atoms/Modal/Modal.js

```diff
@@ -74,21 +74,41 @@ class Modal extends PureComponent<ModalProps, ModalState> {
 
   openModal() {
     if (!this.state.isOpen) {
+      Modal.openedModals += 1;
+
       this.setState({ isOpen: true });
 
       if (typeof this.props.onOpen === 'function') {
         this.props.onOpen();
       }
+
+      this.updateBodyClass();
     }
   }
 
   closeModal() {
     if (this.state.isOpen) {
+      Modal.openedModals -= 1;
+
       this.setState({ isOpen: false });
 
       if (typeof this.props.onClose === 'function') {
         this.props.onClose();
       }
+
+      this.updateBodyClass();
+    }
+  }
+
+  updateBodyClass() {
```

Comment:
I think `upadteBodyClass` is meaningless function name. Can you add more specificity for this method?

Reply:
@vorobeez fixed